docker/chromium/tests-syncer/Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN apt-get update && \
 RUN apt-get update && \
     apt-get install -y \
         git \
+	python-is-python3 \
```

Comment:
What magic does this do, in order to fix the problem?

Reply:
there's a chrome script that runs "python". Since python2 was replaced, this doesn't exist on most systems anymore.
Fixing this actually allows the initialization to complete and for syncing to happen instead of being in an init loop.